EIPS/eip-2565.md

```diff
@@ -53,42 +93,11 @@ You can find comparison of these two complexity fomulas for the current test vec
 
 In addition to modifying the `mult_complexity` formula as above, we also recommend wrapping the entire function with a minimum gas price of 100 to ensure that a minimum amount of gas is used when the precompile is called e.g. `max(100,floor(mult_complexity(x)/GQUADDIVISOR))`
 
-### **Recommended** Option (2): Change value of GQUADDIVISOR
-`GQUADDIVISOR` is set to `20` per EIP-198. We recommend changing the value of this parameter to `3` to account for the changes in the recommended 'computational complexity' formula above.
-
-### Option (3): Replace libraries used by ModExp precompiles
-ModExp benchmarks for different libraries can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Fq3d3wUjGN0R_FX-VPj7TKhCK33ac--P4QXB9MPQ8iw/edit?usp=sharing).
+#### Rationale
 
-While alternative libraries can provide improved a further 2-5x improvement in performance, this option is not recommended at this time.
-
-## Rationale
-
-### **Recommended** Option (1): Modify ‘computational complexity’ formula
 A comparison of the current ‘complexity’ function and the proposed function described above can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1-xBzA-2-l2ZQDQ1eh3XXGZjcRSBQ_Hnp7NubXpbiSUY/edit?usp=sharing).
 
 ![Option 1 Graph](../assets/eip-2565/Complexity_Regression.png)
```

Comment:
There are no longer options, so the asset title "Option 1 Graph" no longer makes sense.